### PR TITLE
curation of plugin list in readme

### PR DESCRIPTION
**Pull request by gabemontero**
@bparees ptal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,11 +144,16 @@ $ s2i build https://github.com/your/repository openshift/jenkins-1-centos7 your_
...
+* **OpenShift Client Plugin**
+Visit [the upstream repository](https://github.com/openshift/jenkins-client-plugin) as well as the [Jenkins plugin wiki](https://wiki.jenkins-ci.org/display/JENKINS/OpenShift+Client+Plugin).  With the lessons learned from OpenShift Pipeline Plugin, as well as adjustments to the rapid evolutions of both Jenkins and OpenShift, this experimental plugin, currently included in the Centos images for this repository, is viewed as the long term replacement for OpenShift Pipeline Plugin.
+
+* **OpenShift Sync Plugin**
+Visit [the upstream repository](https://github.com/openshift/jenkins-sync-plugin) as well as the [Jenkins plugin wiki](https://wiki.jenkins-ci.org/display/JENKINS/OpenShift+Sync+Plugin).  A donation from the fabric8io effort, this plugin facilitates the integration between the OpenShift Jenkinsfile Build Strategy and Jenkins Pipelines.  See the [OpenShift documentation](https://docs.openshift.com) for more details.
```
Comment (bparees) [inline README.md]: strike the wording about it being a donation from the fabric8io effort.

Comment (gabemontero) [inline README.md]: update pushed

**Comment by bparees** [comment]
lgtm

**Comment by gabemontero** [comment]
https://ci.openshift.redhat.com/jenkins/job/jenkins/548/console completed successfully

**Pull request merged**